.github/pull_request_template.md

```
@@ -0,0 +1,40 @@
+- [ ] refactor (code restructuring)
+- [ ] perf (performance)
+- [ ] test (tests)
+- [ ] chore (maintenance)
+- [ ] translation (i18n)
+- [ ] ci (CI/CD)
+
+## Testing
+
+<!-- How was this tested? What devices/Android versions? -->
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)